src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.Widget;
+using AndroidX.AppCompat.Widget;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class ImageHandler : ViewHandler<IImage, ImageView>
+	{
+		protected override ImageView CreateNativeView() => new AppCompatImageView(Context);
+
+		public static void MapAspect(ImageHandler handler, IImage image)
+		{
+			handler.NativeView?.UpdateAspect(image);
+		}
+
+		public static void MapIsAnimationPlaying(ImageHandler handler, IImage image)
+		{
+			handler.NativeView?.UpdateIsAnimationPlaying(image);
+		}
+
+		public static async void MapSource(ImageHandler handler, IImage image) =>
```

Comment:
can we call this with some standard fireandforget mechanism?

https://github.com/dotnet/maui/blob/4455126a5fe4cbbd58e23f8c29e6007f9ee0712c/src/Core/src/TaskExtensions.cs

Reply:
and now that you have loggers maybe we can add a generic ILogger one of these

Reply:
just did that lol

Reply:
The issue with that fire and forget is that it just logs an error. However, with the await in here, it throws an exception. What is better?